app/templates/components/promise-item.hbs

```diff
@@ -1,76 +1,76 @@
-{{#list.cell
-  class=(concat "list__cell_main " expandedClass)
-  style=labelStyle
-  on-click=(action toggleExpand model)
+{{#@list.cell
```

Comment:
Was this `@` on purpose? I suppose it is valid, but looks odd having `#@`

Reply:
It looks terrible :-P. But `list` is passed in that way, right? Maybe we should get a consult from an expert?

Reply:
Does angle bracket syntax work here?

Reply:
Yep :+1:

Reply:
Nice! That looks much better 😃